modules/sdk/project-templates/project-templates-form-field/src/test/java/com/liferay/project/templates/form/field/ProjectTemplatesFormFieldTest.java

```diff
@@ -524,18 +524,64 @@ public void testBuildTemplateFormField73() throws Exception {
 		}
 	}
 
+	@Test
+	public void testBuildTemplateFormField73Maven() throws Exception {
```

Comment:
Hey @lawrence-lee, can you please just let me know the need of this new test? I'm not sure if I got it. :grin:

Reply:
I think I already removed this test case and I put it in the main test class. Ahh yes it says outdated!